config/jobs/kubernetes/sig-docs/sig-docs-periodics.yaml

```diff
@@ -0,0 +1,20 @@
+periodics:
+  - name: kubernetes-website-link-checker
+    annotations:
+      testgrid-dashboards: sig-docs-kubernetes-website
+      testgrid-tab-name: website-link-check-result
+    decorate: true
+    interval: 168h # 1 week
+    labels:
+      preset-service-account: "true"
+      preset-k8s-ssh: "true"
+    extra_refs:
+    - org: kubernetes
+      repo: website
+      base_ref: master
+      path_alias: "k8s.io/website"
+    spec:
+      containers:
+        - image: irvifa/kubernetes-hugo:1.0
+          command: ["/bin/sh"]
+          args: ["-c", "hugo --config config.toml,linkcheck-config.toml --buildFuture && htmltest"]
```

Comment:
How are the results captured?
Is it correct that the `htmltest` link checker only generates results for the `en` content?
Could the docs job also run the new python link checker and report these results? The python checker could be updated to return a pass/fail and the output forwarded to a `testgrid` or adapted to render HTML (or another format).